src/components/calcite-flow/calcite-flow.tsx

```diff
@@ -4,6 +4,9 @@ import { CSS } from "./resources";
 
 import { CalciteTheme, FlowDirection } from "../interfaces";
 
+/**
+ * @slot - A slot for adding `calcite-flow-items` to the flow.
```

Comment:
Nitpick: `` `calcite-flow-items` `` -> `` `calcite-flow-item`s ``

☝️ applies to other pluralized components.